device name path changed, Issue #597

juhlee-microsoft opened the pull request:
Below command returns two lines now.

find /sys/devices/* -name "*${synthetic_interface}" | grep "pci" | sed 's/\// /g'| awk '{print $12}'

The actual path shows like below with the command, find /sys/devices/* -name "*eth0"
/sys/devices/LNXSYSTM:00/LNXSYBUS:00/ACPI0004:00/VMBUS:00/0ce8bfb4-39c0-4431-9830-49a806c94fad/pci9830:00/9830:00:02.0/net/**eth1**/upper_eth0
/sys/devices/LNXSYSTM:00/LNXSYBUS:00/ACPI0004:00/VMBUS:00/000d3aa2-81af-000d-3aa2-81af000d3aa2/net/eth0

This should be 'print $11'

The issue #597

lizzha commented on Testscripts/Windows/GOLANG-BENCHMARK-TEST.ps1:
```diff
@@ -0,0 +1,107 @@
+}
+
+# Main Body
+Main -allVMData $AllVmData -CurrentTestData $CurrentTestData
```
no parameter defined in Main function

lizzha commented on XML/TestCases/Other.xml:
```diff
@@ -13,4 +13,20 @@
+            <param>GOLANG_BENCHMARK_CODE=https://partnerpipelineshare.blob.core.windows.net/binarytools/golang_benchmark.tar.gz</param>
+            <param>GOLANG_INSTALL_PACKAGE=https://partnerpipelineshare.blob.core.windows.net/binarytools/go1.11.4.linux-amd64.tar.gz</param>
+        </TestParameters>
+        <Category>Benchmark</Category>
```
should it be put in Performance category?

xian123 replied:
Put it in Performance category.

lizzha commented on XML/TestCases/Other.xml:
```diff
@@ -13,4 +13,20 @@
+        </TestParameters>
+        <Category>Benchmark</Category>
+        <Area>Golang</Area>
+        <Tags>benchmark,golang</Tags>
```
remove the benchmark in tags

juhlee-microsoft committed:
device name path changed

Below command returns two lines now.

find /sys/devices/* -name "*${synthetic_interface}" | grep "pci" | sed 's/\// /g'| awk '{print $12}'

The actual path shows like below with the command, find /sys/devices/* -name "*eth0"
/sys/devices/LNXSYSTM:00/LNXSYBUS:00/ACPI0004:00/VMBUS:00/0ce8bfb4-39c0-4431-9830-49a806c94fad/pci9830:00/9830:00:02.0/net/eth1/upper_eth0
/sys/devices/LNXSYSTM:00/LNXSYBUS:00/ACPI0004:00/VMBUS:00/000d3aa2-81af-000d-3aa2-81af000d3aa2/net/eth0

This should be 'print $11'

juhlee-microsoft committed:
Pass VM generation info to sriov_constants.sh

VM Gen info required in bash execution

juhlee-microsoft committed:
Branch out blocks per VM gen condition

juhlee-microsoft committed:
Fxied the typo

juhlee-microsoft committed:
Fixed the file overwriting issue
